src/types/ScreenInfoUiaProviderBase.h

```diff
@@ -105,7 +105,7 @@ namespace Microsoft::Console::Types
         // weak reference to IUiaData
         IUiaData* _pData;
 
-        std::wstring _wordDelimiters;
+        std::wstring_view _wordDelimiters;

```

Comment:
Actually, no, this one is quite bad. Always use strings for members _unless_ you can be 100% certain that the memory pointed to by a string view will _not be released_.

A wstring_view is a read-only view at somebody else's memory.